iron-request.html

```diff
@@ -248,7 +249,13 @@
         'xml': 'application/xml',
         'arraybuffer': 'application/octet-stream'
       }[options.handleAs];
-      var headers = options.headers || {};
+      var headers = options.headers || Object.create(null);
```

Comment:
You'll get more predictable behavior by opting out of Object's prototype for map-like objects. e.g. this one caused me no end of grief (and ended up being a bug in the fullproof indexer): (try it in both Chrome and Firefox)

```javascript
var o = {}; 
if (o.watch) { 
  console.log('surely not...'); 
}
```

Hopefully there will never be a clash between a lowercase header name and a value on Object prototype, but there's no real reason to chance it.